pages/common/rg.md

```diff
@@ -1,6 +1,6 @@
 # ripgrep
 
-> A fast command-line search tool.
+> A command-line search tool.
```

Comment:
Another instance of adjective removal resulting in a lack of detail, in my opinion. What about this?
```suggestion
> A recursive line-oriented CLI search tool.
> Aims to be a faster alternative to `grep`.
```